packages/quantic/cypress/page-objects/configurator.ts

```diff
@@ -7,7 +7,7 @@ const getInputSelector = (field: string): string =>
   `lightning-input[data-cy="cfg-${field}"] input`;
 
 export const configure = (
-  options: Record<string, string | number | boolean> = {}
+  options: Record<string, string | number | boolean | object> = {}
```

Comment:
The problem I see allowing an `object` here is that the form cannot be used by a human anymore (unless a user can enter JSON that is automatically deserialized).

Reply:
Now I accept a string than I parse it to object